lib/komponent/core/component_helper.rb

```diff
@@ -7,5 +7,10 @@ def property(name, options = {})
 
   def self.extended(component)
     component.properties = {}
+    component.class_eval do
+      def block_given_to_component?
+        !!@block_given_to_component
```

Comment:
`@block_given_to_component.present?`

Reply:
`.present` is a method provided by `ActiveSupport`. Its not part of ruby core.

Reply:
So? Komponent is always used with Rails, right?